packages/frontend/src/features/allertsPanel/components/AllertsSection.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import styled from "styled-components";
+
+import AllertCell from "./AllertCell";
+import ScrollBar from "./ScrollBar"
+
+const Wrapper = styled.section`
+    position: relative;
+    height: 666px;
```

Comment:
raczej słaby pomysł ustawiać wysokość na sztywno, zwłaszcza tak duzą, lepiej to zrobić tak by samo sie rozciągało/ dopasowywało